### PR TITLE
lib: Add support for multiple interrupts per node

### PR DESCRIPTION
**Pull request by nfcampos**
- Includes support for interrupt loops

### CODE REVIEW

File: libs/langgraph/langgraph/pregel/algo.py [hunk 1]
```diff
@@ -37,13 +37,13 @@
     CONFIG_KEY_CHECKPOINT_NS,
     CONFIG_KEY_CHECKPOINTER,
     CONFIG_KEY_READ,
-    CONFIG_KEY_RESUME_VALUE,
```
Comment (hinthornw) [inline libs/langgraph/langgraph/pregel/algo.py:40]: Is CONFIG_KEY_RESUME_VALUE used anymore?

File: libs/langgraph/langgraph/pregel/loop.py [hunk 2]
```diff
@@ -263,8 +264,28 @@ def put_writes(self, task_id: str, writes: Sequence[tuple[str, Any]]) -> None:
         """Put writes for a task, to be read by the next tick."""
         if not writes:
             return
+        # deduplicate writes to special channels, last write wins
```
Comment (hinthornw) [inline libs/langgraph/langgraph/pregel/loop.py:267]: ooc: why do we know we don't have to deduplicate if one of the writes is to a regular channel?

File: libs/langgraph/langgraph/pregel/algo.py (hunk 1 above)
Comment (nfcampos) [reply libs/langgraph/langgraph/pregel/algo.py:40]: No, removed now

File: libs/langgraph/langgraph/pregel/loop.py (hunk 2 above)
Comment (nfcampos) [reply libs/langgraph/langgraph/pregel/loop.py:267]: Regular channels can't be written to more than once (writes after the first are ignored)

**Pull request merged**